kano_updater/ui/install_window.py

```diff
@@ -54,6 +62,29 @@ def __init__(self):
 
         self._start_install()
 
+    def _launch_game(self, window=None, event=None):
+        try:
+            if event and event.get_keycode()[1] == 44:  # [J] key
```

Comment:
Ah, I've looked for this and did not find it! Thank you so much, will put it in now! :)